test/zimwriterfs-zimcreatorfs.cpp

```diff
@@ -127,3 +128,49 @@ TEST(ZimCreatorFSTest, ThrowsErrorIfDirectoryNotExist)
     ZimCreatorFS zimCreator("Non-existing-dir");
   }, std::invalid_argument );
 }
+
+struct Redirect {
+  std::string path;
+  std::string title;
+  std::string target;
+};
```

Comment:
Why don't you put this in `zimcreatorfs.h` and replace the three parameters of the redirect handler with a single one? It also makes sense to declare `parse_redirectArticles()` in `zimcreatorfs.h` too.

Reply:
Done